scapy/layers/inet.py

```diff
@@ -1645,8 +1645,16 @@ def graph(self, ASres=conf.AS_resolver, padding=0, **kargs):
 @conf.commands.register
 def traceroute(target, dport=80, minttl=1, maxttl=30, sport=RandShort(), l4=None, filter=None, timeout=2, verbose=None, **kargs):  # noqa: E501
     """Instant TCP traceroute
-traceroute(target, [maxttl=30,] [dport=80,] [sport=80,] [verbose=conf.verb]) -> None  # noqa: E501
-"""
+
+       target:  hostnames or IP addresses
```

Comment:
Would you mind using the 
`:param target: ...`
Format ?

Otherwise good to go !

Reply:
Done!